### PR TITLE
[Typescript] Added missing bar props & canvas calendar definition

### PR DESCRIPTION
**Pull request by marnett-git**
Added missing colorBy and borderColor property types to nivo/bar.

An empty object is defined for borderColor - which works but may not be correct.

### CODE REVIEW

File: packages/bar/index.d.ts [hunk 1]
```diff
@@ -113,6 +113,8 @@ declare module '@nivo/bar' {
         labelTextColor: InheritedColorProp<BarDatumWithColor>
 
         colors: OrdinalColorsInstruction
+		colorBy: OrdinalColorsInstruction
+		borderColor: {}
```
Comment (marnett-git) [inline packages/bar/index.d.ts]: An empty object defined works, but may not be correct solution.

**Comment by pedroapfilho** [comment]
@marnett-git it seems that your PR has some formatting "errors", could you check it out?

**Comment by marnett-git** [comment]
@pedroapfilho Fixed

**Comment by marnett-git** [comment]
I don't know why the latest commit is failing the formatting check - all i did was add the required white-space.
If someone could review this that'd be appreciated thanks.

**Comment by wyze** [comment]
I went to try and fix the formatting but I messed up and accidentally merged this in. I am keeping the calendar changes but the `colorBy` prop is going away so I removed that part of the commit. 

Thanks @marnett-git!

**Pull request merged**